features/step_definitions/oauth.rb

```diff
@@ -47,10 +47,11 @@
 Given /^authentication successfully rolls out after config changes$/ do
   ensure_admin_tagged
   interval_time = 5
-  timeout = 400 # set 400 seconds here due to https://bugzilla.redhat.com/show_bug.cgi?id=1958198, after the bug fixed, the seconds should be reduced accordingly
+  processingtimeout = 400 # set 400 seconds here due to https://bugzilla.redhat.com/show_bug.cgi?id=1958198, after the bug fixed, the seconds should be reduced accordingly
```

Comment:
Could you reduce processingtimeout, and update the comment (like if too long open new bug like 1958198)?

Reply:
`timeout` is a good variable name here, do we need to rename it? It's very readable to a person that doesn't understand the technics underneath the OAuth operator(like me).